app/views/api/services/forms/_integration_settings.html.slim

```diff
@@ -1,150 +1,29 @@
-javascript:
-  $(document).ready(function(){
-    // warning when changing auth mode
-    var auth_mode_changed = false;
-    var current_auth_mode = "#{@service.proxy.authentication_method}";
+= javascript_pack_tag 'settings'
 
-    $("input[name='service[backend_version]']").change( function(){
-      auth_mode_changed = true;
-    });
+- @service.deployment_option ||= 'hosted'
```

Comment:
Not sure if I like changing the state of `@service` in a view. Can we use a helper, or a decorator that overrides `Service#deployment_option` with something like `super || 'hosted'`?

Reply:
OK I'm not sure why this is needed but I moved it the closest where it is used, in `integrations_helper`.